docs/api-reference/next/image.md

```diff
@@ -144,30 +116,32 @@ const MyImage = (props) => {
 
 ### sizes
 
-A string mapping media queries to device sizes. Defaults to `100vw`.
+A string which provides information about how wide the image will be at different breakpoints. Defaults to `100vw`--the full width of the screen.
+
+The sizes property is very important for performance when using `layout="responsive"` or `layout="fill"` with images that will take up less than the full viewport width.
 
-We recommend setting `sizes` when using `layout="responsive"` or `layout="fill"` and your image will **not** be the same width as the viewport.
+For a simple example, if you are using `layout="fill"`, and you know the image will always be less than half the width of the viewport, include the `sizes="50vw"` property. Failing to do so will result in the image being erroneously sent at twice the necessary resolution.
 
 [Learn more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes).
 
 ### quality
 
-The quality of the optimized image, an integer between `1` and `100` where `100`
-is the best quality. Defaults to `75`.
+The quality of the optimized image, an integer between `1` and `100` where `100` is the best quality. Defaults to `75`.
 
 ### priority
 
 When true, the image will be considered high priority and
-[preload](https://web.dev/preload-responsive-images/).
+[preload](https://web.dev/preload-responsive-images/). Lazy loading will also be automatically disabled for all priority images.
 
-Should only be used when the image is visible above the fold. Defaults to
-`false`.
+You should use the `priority` attribute on any image which you suspect will be the [LCP element](https://web.dev/lcp/#what-elements-are-considered). It may be appropriate to have multiple priority images, as different images may be the LCP element for different viewport sizes.
```

Comment:
```suggestion
You should use the `priority` attribute on any image which you suspect will be the [Largest Contentful Paint (LCP) element](https://nextjs.org/learn/seo/web-performance/lcp). It may be appropriate to have multiple priority images, as different images may be the LCP element for different viewport sizes.
```